overlord/devicestate/devicestate_test.go

```diff
@@ -801,9 +803,17 @@ func (s *deviceMgrSuite) TestDoRequestSerialErrorsOnNoHost(c *C) {
 		c.Skip("cannot run test when http proxy is in use, the error pattern is different")
 	}
 
+	const nonexistant_host = "nowhere.nowhere.test"
```

Comment:
Typo: it should be `nonexistent`

Reply:
Fixed! Apologies for my terrible spelling.